notifications/workflow-email/components/RemainingFailureTable.component

```diff
@@ -0,0 +1,27 @@
+<apex:component controller="TestRunEmailController" access="global">
```

Comment:
seems like a single table here would suffice, but with just a  new column for "first failure"